src/test/pybind/test_rbd.py

```diff
@@ -377,6 +377,28 @@ def test_config_list():
     for option in rbd.config_list(ioctx):
         eq(option['source'], RBD_CONFIG_SOURCE_CONFIG)
 
+@require_new_format()
+def test_pool_stats():
+    rbd = RBD()
+
+    create_image()
+    create_image()
+    create_image()
+    create_image()
+    with Image(ioctx, image_name) as image:
+        image.create_snap('snap')
+        image.resize(0)
+
+    stats = rbd.pool_stats_get(ioctx)
+    eq(stats['image_count'], 4)
+    eq(stats['image_provisioned_bytes'], 3 * IMG_SIZE)
+    eq(stats['image_max_provisioned_bytes'], 4 * IMG_SIZE)
+    eq(stats['image_snap_count'], 1)
+    eq(stats['trash_count'], 0)
+    eq(stats['trash_provisioned_bytes'], 0)
+    eq(stats['trash_max_provisioned_bytes'], 0)
+    eq(stats['trash_snap_count'], 0)
```

Comment:
When running all pybind tests it fails with:
```
======================================================================
FAIL: test_rbd.test_pool_stats
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/mgolub/ceph/ceph.ci/src/test/pybind/test_rbd.py", line 119, in _require_new_format
    return fn(*args, **kwargs)
  File "/home/mgolub/ceph/ceph.ci/src/test/pybind/test_rbd.py", line 397, in test_pool_stats
    eq(stats['trash_count'], 0)
AssertionError: 1L != 0
```
Which is due to `TestTrash.test_remove_denied` not doing necessary cleanup. Anyway, probably a good idea to additionally run `rbd.pool_stats_get` before creating the images to make sure the pool is empty?

And you need to do the images cleanup too, because after fixing `TestTrash.test_remove_denied` by adding
 `RBD().trash_remove(ioctx, image_id, True)` I start to observe:
```
======================================================================
FAIL: test_rbd.test_list_empty
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/ssd/ceph/ceph.ci/src/test/pybind/test_rbd.py", line 314, in test_list_empty
    eq([], RBD().list(ioctx))
AssertionError: [] != [u'image175', u'image176', u'image177', u'image178']
```

Reply:
@trociny Sorry about that -- should be better now